tutorials/xr/ar_passthrough.rst

```diff
@@ -0,0 +1,178 @@
+.. _doc_openxr_passthrough:
+
+AR / Passthrough
+================
+
+Augmented Reality is supported through various methods depending on the capabilities of the hardware.
+
+Headsets such as the Magic Leap and glasses such as TiltFive show the rendered result on transparent displays
+allowing the user to see the real world.
+
+Headsets such as the Quest, HTC Elite and Lynx R1 implement this through a technique called passthrough,
+where cameras record the real world and these images are used as the background on top of which our rendered
+result is used.
+
+.. note::
+
+  The way specifically passthrough is implemented has vast differences between different platforms.
+
+  In Godot 4.3 we have implemented a unified approach that is explained on this help page
+  so you don't need to worry about these differences, the :ref:`XRInterface <class_xrinterface>`
+  implementation is now responsible for applying the correct platform dependent method.
+
+  For headsets such as the Meta Quest and HTC Elite you will need to use the OpenXR vendors plugin v3.0.0
```

Comment:
Should `OpenXR vendors plugin` be a link?

Reply:
Done :)